app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts.includes([:comments]).order(created_at: :asc)
+    @posts = @user.posts.order(created_at: :asc)
```

Comment:
- [ ] Great job 👏🏻 team, you did an awesome job, all the tests are passing very well, and I can see you fulfilled all requirements except the `n+1` problem. I would like to tell you that you used `includes` to fix the `n+1` problem but then you removed it, I can see that the previous implementation, to fix the `N+1` problem was not correct. The reason behind fixing the `N+1` problem is to optimize the number of database requests. Because you are working on the local system, it seems fine, but consider the big picture, you build a platform and a million people are using that platform, can you imagine a simple data fetching request, which needs only 1 request to fetch the `post` details and all related `likes` and `comment` data, but you are making a separate request to fetch the post related data, for once it is fine but **N** thousand multiply by 3 requests is far greater than exactly **N** thousand request. Kindly adhere to the project requirement and use `includes` on the controllers wherever it is possible.